integration/inspektor-gadget/integration_test.go

```diff
@@ -148,7 +146,8 @@ func testMain(m *testing.M) int {
 		cleanupCommands = append(cleanupCommands, CleanupInspektorGadget)
 	}
 
-	if !*doNotDeploySPO {
+	deploySPO := !CheckNamespace("security-profiles-operator")
+	if deploySPO {
```

Comment:
I frequently run tests that don't need the SPO. So, I run:

```bash
$ INTEGRATION_TESTS_PARAMS="-run TestExecsnoop -v -no-deploy-spo" make integration-tests
```

How can I do this now?